### DISCUSSION TITLE
Is there a max number of favourites?

### DISCUSSION THREAD

**Discussion by an1uk** [Chouffy/home_assistant_tgtg#59] [Q&A] [answered]
Just wondering is there a max number of favourites? Think I'm only seeing 20 imported.

Seems I can't delete this post, but found my answer here  thanks https://github.com/Chouffy/home_assistant_tgtg/issues/29

**Comment by Chouffy** [comment, marked as answer]
Indeed see #29  - if anybody has time to do it, it would be very appreciated
